test/TesterInternal/SchedulerTests/OrleansTaskSchedulerAdvancedTests_Set2.cs

```diff
@@ -604,19 +604,32 @@ public void ActivationSched_SubTaskExecutionSequencing()
             {
```

Comment:
Above here we have
```C#
                taskChainEnds[chainNum] = task.ContinueWith(t =>
                {
                    if (t.IsFaulted) throw t.Exception;
                    output.WriteLine("Inside Chain {0} Final Task", chainNum);
                    resultHandles[chainNum].SetResult(true);
                }, scheduler);
```

Should we also call `resultHandles[chainNum]..SetException(t.Exception)` in the `t.IsFaulted` case? To speed failures along.